pkg/users/account-details.js

```diff
@@ -98,16 +98,27 @@ function get_expire(name) {
 
 export function AccountDetails({ accounts, groups, current_user, user, shells }) {
     const [expiration, setExpiration] = useState(null);
+    const [path, setPath] = useState("");
+
+    useEffect(() => {
+        const lastlog_version = cockpit.spawn(["test", "-e", "/var/run/utmp"], { err: "ignore" });
+        lastlog_version.then(() => {
+            setPath("/var/run/utmp");
+        }).catch(() => {
+            setPath("/var/lib/lastlog/lastlog2.db");
```

Comment:
It'd feel better to actually check that this exists first.

Reply:
the previous logic didn't check for the existence of `/var/run/utmp`, I'll add a check for `/var/lib/lastlog/lastlog2.db` existing